modules/services/servers/src/main/java/au/com/shiftyjelly/pocketcasts/servers/di/ServersModule.kt

```diff
@@ -81,9 +81,12 @@ class ServersModule {
             val originalResponse = chain.proceed(request)
             var responseBuilder = originalResponse.newBuilder()
             if (request.cacheControl.noCache) {
-                responseBuilder = responseBuilder.header(HEADER_CACHE_CONTROL, "public, max-age=$CACHE_FIVE_MINUTES")
+                responseBuilder = responseBuilder
+                    .header(HEADER_CACHE_CONTROL, "public, max-age=$CACHE_FIVE_MINUTES")
+                    .removeHeader("Pragma")
             } else if (request.cacheControl.onlyIfCached) {
                 responseBuilder.header(HEADER_CACHE_CONTROL, "public, only-if-cached, max-stale=${request.cacheControl.maxStaleSeconds}")
+                    .removeHeader("Pragma")
```

Comment:
I'm not sure what effects it might have on other requests. Shouldn't we do this only for the transcript requests?

Reply:
`TranscriptsManagerImpl` uses `TranscriptCacheServer` to load only transcript URLs. They do not have a specific format.

Reply:
But if you don't think we need I'm happy to approve this PR.

Reply:
Thanks for the snippet! ❤️ I like how you added a custom header to identify such requests.
We only use the transcript interceptor for transcript URLs, so we don't need it. I added a comment in 61f0978 to make it more explicit.